src/main/java/com/ocelotslovebirds/birdhaus/blocks/BirdhouseBlockEntity.java

```diff
@@ -25,7 +32,8 @@
     private final ItemStackHandler itemHandler = createHandler();
     private final LazyOptional<IItemHandler> handler = LazyOptional.of(() -> itemHandler);
 
-    private int counter;
+    private Ticker tickerForBirdSpawns = new FixedIntervalTicker(150);
+    private Ticker tickerForSeedConsumption = new FixedIntervalTicker(100);
```

Comment:
Handling the ticking using a helper class seemed to be a good idea (separation of concerns)